src/main/java/de/tum/cit/aet/artemis/assessment/service/ResultService.java

```diff
@@ -618,9 +622,8 @@ public FeedbackAnalysisResponseDTO getFeedbackDetailsOnPage(long exerciseId, Fee
                 maxOccurrence, filterErrorCategories, pageable);
 
         // 10. Process and map feedback details, calculating relative count and assigning task names
-        List<FeedbackDetailDTO> processedDetails = feedbackDetailPage.getContent().stream().map(detail -> new FeedbackDetailDTO(detail.count(),
+        List<FeedbackDetailDTO> processedDetails = feedbackDetailPage.getContent().stream().map(detail -> new FeedbackDetailDTO(detail.concatenatedFeedbackIds(), detail.count(),
```

Comment:
I think it would be fine to just convert the concatenated string here, it seems like a better solution than just using a concatenated list that always needs to be converted before using it.

If I am not mistaken this is the opposite change request to what @JohannesStoehr wanted to have? I guess we should have a short exchange on this today as we will be onsite anyways :)